packages/medusa/src/models/line-item.ts

```diff
@@ -207,120 +236,133 @@ export class LineItem extends BaseEntity {
  *     type: array
  *     items:
  *       $ref: "#/components/schemas/LineItemAdjustment"
+ *   original_item_id:
+ *     description: The id of the original line item
+ *     nullable: true
+ *     type: string
+ *   order_edit_id:
+ *     description: The ID of the order edit to which a cloned item belongs
+ *     nullable: true
+ *     type: string
+ *   order_edit:
+ *     description: The order edit joined. Available if the relation `order_edit` is expanded.
+ *     nullable: true
+ *     $ref: "#/components/schemas/OrderEdit"
  *   title:
- *     description: "The title of the Line Item, this should be easily identifiable by the Customer."
+ *     description: The title of the Line Item, this should be easily identifiable by the Customer.
  *     type: string
  *     example: Medusa Coffee Mug
  *   description:
- *     description: "A more detailed description of the contents of the Line Item."
+ *     description: A more detailed description of the contents of the Line Item.
+ *     nullable: true
  *     type: string
  *     example: One Size
  *   thumbnail:
- *     description: "A URL string to a small image of the contents of the Line Item."
+ *     description: A URL string to a small image of the contents of the Line Item.
+ *     nullable: true
  *     type: string
  *     format: uri
  *     example: https://medusa-public-images.s3.eu-west-1.amazonaws.com/coffee-mug.png
  *   is_return:
- *     description: "Is the item being returned"
+ *     description: Is the item being returned
  *     type: boolean
- *     example: false
+ *     default: false
  *   is_giftcard:
- *     description: "Flag to indicate if the Line Item is a Gift Card."
+ *     description: Flag to indicate if the Line Item is a Gift Card.
  *     type: boolean
- *     example: false
+ *     default: false
  *   should_merge:
- *     description: "Flag to indicate if new Line Items with the same variant should be merged or added as an additional Line Item."
+ *     description: Flag to indicate if new Line Items with the same variant should be merged or added as an additional Line Item.
  *     type: boolean
- *     example: false
+ *     default: true
  *   allow_discounts:
- *     description: "Flag to indicate if the Line Item should be included when doing discount calculations."
+ *     description: Flag to indicate if the Line Item should be included when doing discount calculations.
  *     type: boolean
- *     example: false
+ *     default: false
```

Comment:
default should be `true`.

Reply:
Good catch. Done in e31492da48e1d04b745c5c39e465fb9583694a63